plugins/services/src/js/filters/ServiceAttribIsFilter.js

```diff
@@ -0,0 +1,45 @@
+import DSLFilterTypes from '../../../../../src/js/constants/DSLFilterTypes';
+import DSLFilter from '../../../../../src/js/structs/DSLFilter';
+import ServiceStatus from '../constants/ServiceStatus';
+
+const TEXT_TO_INSTANCE = {
+  delayed: ServiceStatus.DELAYED,
+  deploying: ServiceStatus.DEPLOYING,
+  na: ServiceStatus.NA,
+  running: ServiceStatus.RUNNING,
+  suspended: ServiceStatus.SUSPENDED,
+  waiting: ServiceStatus.WAITING
+};
+
+/**
+ * This filter handles the `is:status` for services using `getServiceStatus`
+ */
+class ServiceAttribIsFilter extends DSLFilter {
+
+  /**
+   * Handle all `is:XXXX` attrib filters that we can handle.
+   *
+   * @override
+   */
+  filterCanHandle(filterType, filterArguments) {
+    return filterType === DSLFilterTypes.ATTRIB &&
+           filterArguments.label === 'is' &&
```

Comment:
I know it's just being used once, but let's make the code even cleaner by introducing a constant for `is`! ✨